Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -180,24 +177,6 @@ bool GameListItem::IsValid() const
   return true;
 }
 
-void GameListItem::ReloadINI()
-{
-  if (!IsValid())
-    return;
-
-  IniFile ini = SConfig::LoadGameIni(m_game_id, m_Revision);
-  ini.GetIfExists("EmuState", "EmulationStateId", &m_emu_state, 0);
-  ini.GetIfExists("EmuState", "EmulationIssues", &m_issues, std::string());
-
-  m_custom_name.clear();
-  m_has_custom_name = ini.GetIfExists("EmuState", "Title", &m_custom_name);
```

Comment:
Actually... I haven't thought of it until now, but the rename-through-INI feature is ignored if there's a custom name from titles.txt/wiitdb.txt, and we ship with a huge wiitdb.txt nowadays, so I suppose this feature already isn't working... Since people haven't complained about it, I guess it's fine if we drop it?